src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmrc.api.service
+
+import play.api.libs.json.Json
+import play.api.libs.ws.StandaloneWSRequest
+import uk.gov.hmrc.api.client.HttpClient
+import uk.gov.hmrc.api.conf.TestConfiguration
+import uk.gov.hmrc.api.utils.JsonGetter.getJsonFile
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+class UknwAuthCheckerApiService extends HttpClient {
+
+  val authorisationsUrl: String = TestConfiguration.url("uknw-auth-checker-api")
+
+  def authorisations(authToken: String): StandaloneWSRequest#Self#Response = {
+    val individualPayload = getJsonFile("authRequest200_single.json")
```

Comment:
loading this here does not seem wise. Should be in the spec.

Reply:
Refactored code to send the the test data from Spec